contracts/programs/gumball-machine/src/lib.rs

```diff
@@ -106,6 +114,9 @@ pub struct DispenseSol<'info> {
     /// This key must sign for all write operations to the NFT Metadata stored in the Merkle slab
     #[account(mut)]
     bubblegum_authority: AccountInfo<'info>,
+    /// CHECK: PDA is checked in Bubblegum
+    #[account(mut)]
+    bubblegum_mint_request: AccountInfo<'info>,
```

Comment:
nit: maybe `willy_wonka_mint_request` is a better name because it's `willy_wonka`'s unique mint request (which is the mint request account for this machine), rather than the request of the `bubblegum_authority` for instance